.github/workflows/pulp_images.yml

```diff
@@ -103,8 +103,12 @@ jobs:
         run: |
           podman version
           buildah version
-          podman build --format docker --file images/Containerfile.core.base --tag pulp/base:${TEMP_BASE_TAG} .
-          podman build --format docker --file images/pulp_ci_centos/Containerfile --tag pulp/pulp-ci-centos:${TEMP_BASE_TAG} --build-arg FROM_TAG=${TEMP_BASE_TAG} .
+          sudo podman run --rm --privileged multiarch/qemu-user-static --reset -p yes
+          for ARCH in arm64 amd64
```

Comment:
Can we have different workflows/steps for each arch? So we don't have to use more bash magic here...

Reply:
@StopMotionCuber I agree, let's stick with this for now.